packages/angular_devkit/schematics/BUILD

```diff
@@ -13,13 +13,36 @@ load("@build_bazel_rules_nodejs//:defs.bzl", "npm_package")
 
 licenses(["notice"])  # MIT License
 
+# @angular-devkit/schematics/src/formats
```

Comment:
Why is this needed? `standardFormats` is already available as part of the public API of `@angular-devkit/schematics` in the `formats` namespace

You can you it like;
```ts
import { formats } from '@angular-devkit/schematics';
formats.standardFormats
```